test/endpoint_test.go

```diff
@@ -242,7 +242,7 @@ func TestPostCallbackToken(t *testing.T) {
 
 			require.Equal(t, http.StatusOK, response.StatusCode())
 			require.NotNil(t, response.JSON200)
-			require.Equal(t, Status_SEEN_ON_NETWORK, response.JSON200.TxStatus)
+			require.Equal(t, Status_ACCEPTED_BY_NETWORK, response.JSON200.TxStatus)
```

Comment:
Hey @shotasilagadzetaal 
I don't think the solution should be to expect now another status everywhere. IMO it should work the same.

Here is an alternative approach how this can be achieved by sending the request 1 or 2 seconds later: https://github.com/bitcoin-sv/arc/commit/ae2d53de08991301d2e061201c881b3138b11d26

Reply:
Reverted